.prettierrc

```diff
@@ -1,5 +1,7 @@
 {
   "singleQuote": true,
   "trailingComma": "es5",
-  "semi": true
+  "semi": true,
+  "printWidth": 100,
+  "arrowParens": "avoid"
```

Comment:
I like always having arrow parens :/

Reply:
Our current code style omits arrow parentheses. See eslint-config-heydoctor.

Reply:
in typescript you need arrow parents for type annotations so does this even make sense anymore?

Reply:
Arrow function infer param types when used in a `map` or `filter`.

Reply:
im arguing that since arrow parens are used everywhere but those specific instances we might want to reconsider this style altogether. can be a discussion outside this PR

Reply:
yes we can reconsider our style guide but this PR is for adding TS to synaptik 😄 Would appreciate feedback on my approach instead of the style 😃

Reply:
I've seen, hence " :/ "